tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -754,7 +754,7 @@ public function updateLastShareWithSettings(
 		$user = $this->getActualUsername($user);
 
 		if ($updateLastPublicLink) {
-			$share_id = (string) $this->getLastCreatedPublicShare()->id;
+			$share_id = (FeatureContext::isUsingSharingNG()) ? $this->shareNgGetLastCreatedLinkShareID() : (string) $this->getLastCreatedPublicShare()->id;
```

Comment:
`isUsingSharingNG` is not a static method
```suggestion
			$share_id = ($this->featureContext::isUsingSharingNG()) ? $this->shareNgGetLastCreatedLinkShareID() : (string) $this->getLastCreatedPublicShare()->id;
```